pkg/tests/BUILD

```diff
@@ -536,3 +536,22 @@ filegroup(
     ],
     visibility = ["//visibility:public"],
 )
+
+pkg_tar(
+    name = "stamped_tar",
+    srcs = ["BUILD"],
+    stamp = 1,
+)
+
+py_test(
+    name = "stamp_test",
+    srcs = [
+        "stamp_test.py",
+    ],
+    data = [
+        "stamped_tar.tar",
+    ],
+    deps = [
+        "@bazel_tools//tools/python/runfiles",
+    ],
+)
```

Comment:
Perhaps it would also be worth testing `stamp = -1` with a transition on `--stamp`.  Is this even possible?